jdbc-driver/src/main/java/com/google/cloud/dataproc/jdbc/DataprocDriver.java

```diff
@@ -29,16 +31,17 @@
 public class DataprocDriver implements Driver {
 
     // Expected JDBC URL prefix format
-    private static final String URL_PREFIX = "jdbc:dataproc://";
+    public static final String DATAPROC_JDBC_HIVE_URL_SCHEMA = "jdbc:dataproc://hive/";
 
     @Override
     public Connection connect(String url, Properties info) throws SQLException {
         return acceptsURL(url) ? createConnection(url, info) : null;
     }
 
     @Override
-    public boolean acceptsURL(String url) throws SQLException {
-        return url != null && Pattern.matches(URL_PREFIX + ".*", url);
+    public boolean acceptsURL(String url) {
+        // Other protocols may be supported later, but for now only accept Hive
+        return url != null && Pattern.matches(DATAPROC_JDBC_HIVE_URL_SCHEMA + ".*", url);
```

Comment:
Seems like you are checking if `url` starts from the prefix, if this is the case, then it will be simpler and more efficient to use `startsWith` method:
```java
url.startsWith(DATAPROC_JDBC_HIVE_URL_SCHEMA);
```